apartment/elastic/queries.py

```diff
@@ -18,6 +18,10 @@ def get_apartments(project_uuid=None):
     return response
 
 
+def get_project(project_uuid=None):
+    return get_projects(project_uuid)
```

Comment:
I agree with this comment.

Reply:
The naming is the problem here. `get_projects` has been done a bit incorrectly as the developer can expect that it returns multiple values but it contains an implementation that can return a single project. That causes again refactoring so I tried to avoid it this time and I instead added a new function called get_project which means better that it return only one project.

Reply:
I think I kind of see where you are coming from, but I also think that this will only make the situation even more confusing. Now you will have also function `get_project()` which can, in turn, actually return a list of projects if used without providing `project_uuid` 🙃 So I would at least make the parameter mandatory in the function.

But I still like the best to just use the original function with the plural name, IMO it returning just one project isn't a big deal, especially when that one is actually returned in a list if I've not mistaken.

But your PR, your call 🙂

Reply:
Changed to use `get_projects()`.